client/src/js/hmm/components/Item.js

```diff
@@ -1,39 +1,25 @@
 import React from "react";
 import PropTypes from "prop-types";
-import { LinkContainer } from "react-router-bootstrap";
 import { keys, map, reject } from "lodash-es";
 import styled from "styled-components";
-import { device, Label, ListGroupItem } from "../../base";
+import { device, Label, LinkBox } from "../../base";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['device' is defined but never used.](https://app.codacy.com/manual/virtool/virtool/pullRequest?prid=5026542)